packages/react-core/src/components/TreeView/TreeViewListItem.tsx

```diff
@@ -197,7 +197,7 @@ const TreeViewListItemBase: React.FunctionComponent<TreeViewListItemProps> = ({
     <li
       id={id}
       className={css(styles.treeViewListItem, internalIsExpanded && styles.modifiers.expanded)}
-      {...(internalIsExpanded && { 'aria-expanded': 'true' })}
+      aria-expanded={internalIsExpanded}
```

Comment:
aria-expanded should always be set with either true or false, not only when true, so this just fixes that.